sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-card>
+    <v-container>
+      <body-header title="History of Bypassed Restrictions" class="m-1">
+        <template #actions>
+          <check-permission-role :role="Role.StudentAddRestriction">
+            <template #="{ notAllowed }">
+              <v-btn
+                @click="addBypassedRestriction"
+                class="float-right"
+                color="primary"
+                data-cy="addRestrictionButton"
+                prepend-icon="fa:fa fa-plus-circle"
+                :disabled="notAllowed"
+                >Bypass A Restriction</v-btn
+              >
+            </template>
+          </check-permission-role>
+        </template>
+      </body-header>
+      <content-group class="mt-4">
+        <toggle-content
+          :toggled="!bypassedRestrictions.length"
+          message="No bypassed restrictions found."
+        >
+          <DataTable
```

Comment:
As per the the technical AC, we should use the vuetify data table and not the primevue. 
![image](https://github.com/user-attachments/assets/890c3f34-5dea-477d-afc4-e94da2dd60e5)

Here is a sample implementation of vuetify data table w/o server pagination [CAS Vuetify Table](https://github.com/bcgov/SIMS/blob/c26b2bde73ad76f101165c9f7863e96f37170f4a/sources/packages/web/src/views/aest/student/CASSupplierInformation.vue#L28)

Reply:
Updated.